source/eprosima-suite/eprosima_suite.rst

```diff
@@ -0,0 +1,191 @@
+  <https://eprosima-shapes-demo.readthedocs.io/>`_.
+
+- :ref:`Fast DDS Monitor <fast_dds_suite_monitor>`: eProsima Fast DDS Monitor is a graphical desktop application aimed
+  at monitoring DDS environments deployed using the *eProsima Fast DDS* library. Thus, the user can monitor in real
+  time the status of publication/subscription communications between DDS entities. They can also choose from a wide
+  variety of communication parameters to be measured (latency, throughput, packet loss, etc.), as well as record and
+  compute in real time statistical measurements on these parameters (mean, variance, standard deviation, etc.).
+
+  You can read more about this application on the `Fast DDS Monitor documentation page
+  <https://fast-dds-monitor.readthedocs.io/>`_.
+
+- :ref:`eProsima Micro XRCE-DDS <micro_xrce_dds>`:
+  eProsima Micro XRCE-DDS is a software solution that allows communicating
+  eXtremely Resource Constrained Environments (XRCEs) with an existing DDS network. This implementation complies with
+  the specification of the `eXtremely Resource Constrained Environments DDS (DDS-XRCE)
+  <https://www.omg.org/spec/DDS-XRCE/>`_. protocol as defined and maintained by the Object Management Group (OMG)
+  consortium.
+
+  You can read more about this application on the
+  `eProsima Micro XRCE-DDS documentation page <https://micro-xrce-dds.docs.eprosima.com/>`_.
+
+To load this image into your Docker repository, from a terminal run
+
+.. code-block:: bash
+
+ $ docker load -i ubuntu-eprosima-suite:<FastDDS-Version>.tar
+
+You can run this Docker container as follows
+
+.. code-block:: bash
+
+ $ xhost local:root
+ $ docker run -it --privileged -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix \
+ ubuntu-eprosima-suite:<FastDDS-Version>
+
+From the resulting Bash Shell you can run each feature.
+
+.. _fast_dds_suite_examples:
+
+Fast DDS Examples
+-----------------
+
+Included in this Docker container is a set of binary examples that showcase several functionalities of the
+Fast DDS libraries. These examples' path can be accessed from a terminal by typing
+
+.. code-block:: bash
+
+ $ goToExamples
+
+From this folder you can access all examples, both for DDS and RTPS. We detail the steps to launch two such
+examples below.
+
+Hello World Example
+^^^^^^^^^^^^^^^^^^^
+
+This is a minimal example that will perform a Publisher/Subscriber match and start sending samples.
+
+.. code-block:: bash
+
+ $ goToExamples
+ $ cd HelloWorldExample/bin
+ $ tmux new-session "./HelloWorldExample publisher 0 1000" \; \
+ split-window "./HelloWorldExample subscriber" \; \
+ select-layout even-vertical
+
+This example is not constrained to the current instance. It's possible to run several instances of this
+container to check the communication between them by running the following from each container.
+
+.. code-block:: bash
+
+ $ goToExamples
+ $ cd HelloWorldExample/bin
+ $ ./HelloWorldExample publisher
+
+or
```

Comment:
I'd rather see something like: "on one terminal, run one image and do this; on another terminal run another instance and do that"